tools/font-icon/src/index.js

```diff
@@ -110,7 +117,8 @@ async function init() {
     },
     tokenPath: '.', 
   }
-  await fs.writeFile(configFile, `module.exports = ${JSON.stringify(config, null, 2)}`, 'utf-8');
+  
+  return _dry ? config : await fs.writeFile(configFile, `module.exports = ${JSON.stringify(config, null, 2)}`, 'utf-8');
```

Comment:
Do we need print config if we use dry mode?

Reply:
The init would return the config in dry run,
How to use the return stuff would up to the user,